### ISSUE TITLE
Trained model

### ISSUE THREAD

**Issue by camgbus** [MIC-DKFZ/nnUNet#28]
Hi Fabian,
thanks a lot for making your code public and taking the time to write comprehensible READMEs.
I'm trying to run your code in a Windows system with GTX1080ti GPUs. The code seems to work correctly after changing the 'path.split("/")[-1]' lines for 'os.path.split(path)[1]', or equivalent for other 'path.split("/")'s.
However, I would like to compare my 2D U-Net model to yours. Could I find anywhere your trained 2D U-Net model for the Hippocampus task?
Alternatively, could I find a plot of your loss while training the 2D U-Net model?
Best,
Camila

**Comment by FabianIsensee** [comment]
Hi,
I can send the model to you, but shouldn't the cross-validation dice be enough? Installing pre-trained models into nnU-Net is possible but I would prefer not to give out lengthy instructions right now. I want to write that up properly in the documentation in the coming weeks.
Best,
Fabian
PS: 2D U-Net is not really good on Hippocampus. 3D U-Net is much better

**Comment by camgbus** [comment]
Hi,
thank you for the reply.
I did get the same Dice results, so I guess it's working correctly. I'm not really interested in learning the Hippocampus dataset but a different on, I just tried the 2D network because I have limited GPU time but wanted to check that I was getting the same results you did.
Don't worry about the instructions for loading the models, I'll wait until they come into the documentation.
Best,
Camila
